gax-java/gax/src/main/java/com/google/api/gax/batching/BatcherStats.java

```diff
@@ -52,6 +54,13 @@ class BatcherStats {
   private final Map<Class, Integer> entryExceptionCounts = new HashMap<>();
   private final Map<Code, Integer> entryStatusCounts = new HashMap<>();
 
+  private final EvictingQueue<String> sampleOfRpcErrors =
+      EvictingQueue.create(
+          Integer.getInteger("com.google.api.gax.batching.errors.max-samples", 50));
```

Comment:
Can I get more info about this system property `com.google.api.gax.batching.errors.max-samples`? Like where is it documented?

Separately, can we extract `Integer.getInteger("com.google.api.gax.batching.errors.max-samples", 50)` to a common field?

Reply:
Added some doc and extracted the constant